setup.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-from distutils.core import setup
+from setuptools import setup
```

Comment:
Do other ROS packages which use `from distutils.core import setup` have to be updated in order to continue to work?